dist/src/main/arctic-bin/conf/config.yaml

```diff
@@ -28,6 +28,7 @@ ams:
     
   self-optimizing:
     commit-thread-count: 10
+    runtime-data-keep-time-millis: 2592000000 # 30d
```

Comment:
I think being accurate to the day is sufficient, and it can be changed to `runtime-data-keep-days`.